scale-tests/sdk.dashboard.py

```diff
@@ -132,29 +97,52 @@ def resource_row(pod_type: str = None) -> G.Row:
     else:
         title = "Resources for all pod types"
 
+    selection = dict(framework_name=FRAMEWORK_VARIABLE)
+    if pod_type:
+        selection["executor_name"] = pod_type
+
     return G.Row(
         title=title,
         repeat=POD_TYPE_VARIABLE.lstrip("$") if pod_type else "",
         panels=[
-            ResourceStat("Pod count", "cpus_limit", pod_type=pod_type).to_single_stat(
-                "count"
+            W.prometheus.PromGraph(
+                data_source=PROMETHEUS_DATA_SOURCE,
+                title="CPU",
+                expressions=[
+                    dict(expr=reduction('sum', metric(m, selection), selection),
+                         legendFormat=title)
+                    for m, title in [('cpus_limit', 'Available')]
+                ],
+                span=3,
+                steppedLine=True,
+                yAxes=G.YAxes(left=G.YAxis(format="short", decimals=0)),
+            ),
+            W.prometheus.PromGraph(
+                data_source=PROMETHEUS_DATA_SOURCE,
+                title="Memory",
+                expressions=[
+                    dict(expr=reduction('sum', metric(m, selection), selection),
+                         legendFormat=title)
+                    for m, title in [('mem_total', 'Used'),
+                                     ('mem_limit', 'Available')]
+                ],
+                span=3,
+                steppedLine=True,
+                yAxes=G.YAxes(left=G.YAxis(format="short", decimals=0)),
```

Comment:
Should we set the units for memory and disk to bytes?

Reply:
OK, done.